09-clustering.Rmd

```diff
@@ -355,42 +367,42 @@ cross-validation with some measure of model prediction error.
 Further, if K is chosen too small, then multiple clusters get grouped together;
 if K is too large, then clusters get subdivided. In both cases, we will potentially miss
 interesting structure in the data. For example, take a look below at the K-means
-clustering of our customer satisfaction and loyalty data for a number of clusters
+clustering of our penguin flipper and bill length data for a number of clusters
 ranging from 1 to 9.
```

Comment:
the figure here doesn't look right. the means don't appear to be positioned correctly within their clusters